src/components/OutboundLink.js

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+var assign = require('react/lib/Object.assign');
+
+var OutboundLink = React.createClass({
+  displayName: 'OutboundLink',
+  statics: {
+    trackLink: function() {
+      console.warn("ga tracking not enabled");
+    }
+  },
+  handleClick: function (e) {
+    e.preventDefault();
+    var props = this.props;
+    OutboundLink.trackLink(props.args, function () {
+      if ( props.target === '_blank' ) {
+        window.open(props.to, '_blank');
```

Comment:
since we only use it in two places, and it's already a var, no real need to pull it out I think. When this code runs it's only going to go one way or the other, so `props.to` is only referenced once in either path, whereas that string is used twice in the same path.